harness/tests/cli/test_user.py

```diff
@@ -9,9 +10,89 @@
 from tests.cli import util
 
 
+def test_user_create_with_password_flag() -> None:
+    with util.standard_cli_rsps() as rsps:
+        userobj = bindings.v1User(active=True, admin=True, username="det-user", id=10017)
+        rsps.post(
+            "http://localhost:8080/api/v1/users",
+            status=200,
+            match=[
+                matchers.json_params_matcher(
+                    params={
+                        "isHashed": True,
+                        "user": {
+                            "username": "test-user-1",
+                        },
+                        "password": api.salt_and_hash("5DCAB140-f49b-4260-a451-fad6a10017ca"),
+                    },
+                    strict_match=False,
+                ),
+            ],
+            json={"user": userobj.to_json()},  # doesn't really match, but doesn't matter yet
+        )
+        cli.main(
+            ["user", "create", "test-user-1", "--password", "5DCAB140-f49b-4260-a451-fad6a10017ca"]
+        )
+
+
+def test_user_create_remote_account() -> None:
```

Comment:
Same comment here as with the passworded one.

Reply:
Eh, this one I do want to keep in as it's relatively simple, and it seems reasonable to try to make sure users aren't prompted for a password (or other weird errors) if they're trying to make a remote-managed user.